transactions/src/main/java/org/ehcache/transactions/xa/internal/XAStore.java

```diff
@@ -656,6 +657,20 @@ public V apply(final K k) {
     private final Map<Store<?, ?>, SoftLockValueCombinedSerializerLifecycleHelper> createdStores = new ConcurrentWeakIdentityHashMap<Store<?, ?>, SoftLockValueCombinedSerializerLifecycleHelper>();
 
     @Override
+    public int rank(final Set<ResourceType> resourceTypes, final Collection<ServiceConfiguration<?>> serviceConfigs) {
```

Comment:
Discussed with @chrisdennis -- unless we need to bundle it with this work, the associated updates will be another PR.